.gitignore

```diff
@@ -1,5 +1,5 @@
 target
 Cargo.lock
 .idea/
-
+rls
```

Comment:
Not a big deal, but I'm curious what caused this to be added?

Reply:
Yeah, I'll clean this up. Was probably some local config I had for rls dumping logs there - I don't think it's useful or valid anymore.